frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx

```diff
@@ -302,12 +343,13 @@ export default class LineAreaBarChart extends Component {
         )}
       >
         {hasTitle && (
-          <ChartCaption
-            series={series}
-            settings={settings}
+          <LegendCaption
+            className="mx1 flex-no-shrink"
+            title={title}
+            description={description}
             icon={headerIcon}
             actionButtons={actionButtons}
-            onChangeCardAndRun={onChangeCardAndRun}
+            onSelectTitle={canSelectTitle ? this.onSelectTitle : undefined}
```

Comment:
This component call is super simple, but others are very complicated and get complicated conditionals inside prop declarations.

For consistency, one idea is to declare the desired prop as a const, before calling the component.

```
const onSelectTitle = canSelectTitle && this.onSelectTitle
```

and later use that when passing the prop

```
…
onSelectTitle={onSelectTitle}
…
```